src/utils.js

```diff
@@ -3,6 +3,21 @@ export default class Utils {
     return Utils.escapedSchema(schema, quote) + Utils.escape((prefix || '') + table, quote);
   }
 
+  static escapeIdentifier(identifier) {
+    if (identifier == null || identifier.length === 0) {
+      return '';
+    }
+
+    const needsQuotes = /[^_A-Z0-9]/i.test(identifier) ||
+                        /^[0-9]/.test(identifier);
+
+    if (needsQuotes) {
+      return '"' + (identifier.replace(/"/g, '""')) + '"';
```

Comment:
Doesn't this turn a single set of " into 2 "s? (`"` => `""`) and then wrap the entire thing in another set of quotes?
Is the objective to escape quotes first? Or remove them?

Reply:
That appears to be correct. It wraps the whole thing in quotes if needsQuotes is true. Then it escapes any interior quotes with the double quotes.

Reply:
Should it just replace with a single `"` then? Running some tests on the browser console produces more quotes than I would expect

Reply:
To escape a double quote in postgres you use 2 double quotes. How many quotes were you expecting?

Reply:
Ah interesting, I didn't know that. TIL